widgets/GCSFile-batchsource.json

```diff
@@ -40,11 +40,21 @@
           }
         },
         {
-          "widget-type": "plugin-list",
+          "widget-type": "select",
           "label": "Format",
           "name": "format",
           "widget-attributes": {
-            "plugin-type": "validatingInputFormat"
+            "values": [
+              "avro",
+              "blob",
+              "csv",
+              "delimited",
+              "json",
+              "parquet",
+              "text",
+              "tsv"
+            ],
+            "default": "text"
```

Comment:
Nit: Do we need a default value? In 6.3 , it forces user to choose a value.

Reply:
This is for 6.1